src/test/java/com/klarna/hiverunner/PreV200HiveShellHiveCliEmulationTest.java

```diff
@@ -29,7 +30,7 @@
 import com.klarna.hiverunner.config.HiveRunnerConfig;
 import com.klarna.hiverunner.sql.cli.hive.PreV200HiveCliEmulator;
 
-//@Ignore TODO: some of these tests fail with Hive 3, should we fix or remove this functionality?
+@Ignore("Disabling the test due to failures when upgrading to Hive 3. This CLI has been deprecated.")
```

Comment:
Let's just delete this class, I don't think it has any value for Hive 3.

Reply:
Thanks for the review! 
I've deleted the test class